lib/abide_dev_utils/xccdf.rb

```diff
@@ -218,6 +229,16 @@ def controls_by_profile_title(profile_title)
         profiles.select { |x| x.title == profile_title }.controls
       end
 
+      def gen_map(opts)
+        os, ver = facter_platform
+        path = opts.fetch(:path, 'data/mappings')
+        mapping_dir = File.expand_path(File.join(path, opts[:type], os, ver))
+        ['title', 'hiera_title', 'hiera_title_num', 'number'].each_with_object({}) do |idx, h|
+          map_file_path = "#{mapping_dir}/#{idx}.yaml"
+          h[map_file_path] = map_indexed(index: idx)
```

Comment:
Why are we not passing `opts[:type]` into `map_indexed()`? It has a `framework` param that accepts the same information.